plugins/velociraptor/src/plugin.cpp

```diff
@@ -351,6 +359,115 @@ class plugin final : public operator_plugin<velociraptor_operator> {
     };
   }
 
+  auto
+  make(invocation inv, session ctx) const -> failure_or<operator_ptr> override {
+    auto args = operator_args{};
+    auto org_id = std::optional<located<std::string>>{};
+    auto request_name = std::optional<located<std::string>>{};
+    auto max_rows = std::optional<located<uint64_t>>{};
+    auto subscribe = std::optional<located<std::string>>{};
+    auto max_wait = std::optional<located<duration>>{};
+    auto query = std::optional<located<std::string>>{};
+    auto profile = std::optional<located<std::string>>{};
+    argument_parser2::operator_("velociraptor")
+      .add("request_name", request_name)
+      .add("org_id", org_id)
+      .add("query", query)
+      .add("max_rows", max_rows)
+      .add("subscribe", subscribe)
+      .add("max_wait", max_wait)
+      .add("profile", profile)
+      .parse(inv, ctx)
+      .ignore();
+
+    if (max_wait && max_wait->inner < 1s) {
+      diagnostic::error("max_wait too low")
```

Comment:
```suggestion
      diagnostic::error("`max_wait` too low")
```

Segments associated with code are always written within backticks.